src/client/game/errors/error.ts

```diff
@@ -10,14 +10,15 @@ const oopsMessages = [
 
 export interface GameError extends Error {
   unrecoverable: boolean;
+  getButton?: (onclick: () => void) => HTMLButtonElement;
 }
 
 export default function showGameError(error: GameError) {
   // Log the error first of all
   console.error(error);
 
   const errDiv = <HTMLDivElement>document.getElementById("gameError");
-
+  if (!errDiv.classList.contains("invisible")) return;
```

Comment:
This is so that we only display one error at at time. All subsequent errors will just be logged to the console until the user dismisses the present one.